lib/i18n/js/formatters/js.rb

```diff
@@ -15,7 +15,8 @@ def format(translations)
         protected
 
         def header
-          %(#{@namespace}.translations || (#{@namespace}.translations = {});\n)
+          text = @import ? %(import I18n from '#{@import}';\n) : ''
```

Comment:
I would suggest to have a `prefix` option instead of import option
So that if someone wants to add more import statements / other JS code, they can add it as well

Reply:
so the yml file would be something like:
```yml
import: "import I18n from 'i18n-js'"

```
right?

Reply:
Yup
no new line required
User is responsible to add newline char / semi-colon